xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/Translations.xml

```diff
@@ -257,16 +257,16 @@ platform.wiki.sheet.desc.secure=If enabled, all this wiki's generated externals
 platform.wiki.sheet.desc.iswikitemplate=Indicate if this wiki can be use as template when creating a new wiki.
 platform.wiki.sheet.desc.server=The alias used to access the wiki.
 platform.wiki.sheet.desc.description=A brief wiki description.
-platform.wiki.sheet.desc.homepage=The home page of the wiki.
+platform.wiki.sheet.desc.homepage=The homepage of the wiki.
 platform.wiki.sheet.prop.wikiprettyname=Pretty name
 platform.wiki.sheet.prop.owner=Owner
 platform.wiki.sheet.prop.secure=Secure (SSL)
 platform.wiki.sheet.prop.description=Description
 platform.wiki.sheet.prop.server=Alias
-platform.wiki.sheet.prop.homepage=Home page
+platform.wiki.sheet.prop.homepage=Homepage
 platform.wiki.sheet.prop.iswikitemplate=Wiki template
 platform.wiki.sheet.descriptor=This page defines how to access the \"{0}\" wiki, and some properties about it. For more configurations, please go to the
-platform.wiki.sheet.descriptor.admin=administraton page of the wiki.
+platform.wiki.sheet.descriptor.admin=Administration page of the wiki.
```

Comment:
Should follow the advice from @tmortagne  on this one and leave the casing (`administration`) as it was initially.

Reply:
Done.